styles.css

```diff
@@ -4,27 +4,54 @@
   box-sizing: border-box;
 }
 
+#store-name {
+  font-family: 'Preahvihear', 'Times New Roman', Times, serif;
+  width: 30rem;
+  padding: 0.25rem;
+  margin: 1rem auto;
+  text-align: center;
+}
+
 form {
-  margin-top: 2rem;
+  font-family: "Roboto", 'Times New Roman', Times, serif;
+  margin: 1rem auto;
   display: flex;
   flex-direction: column;
   width: 15rem;
   height: auto;
-  align-items: flex-start;
+  align-items: flex-end;
   gap: 0.5rem;
   padding: 2rem;
 }
```

Comment:
- [ ] Good job so far, :clap: , However, please ensure that your styling matches   given requirement as much as possible, matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, Please consider the red painting in  your  design :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

![Student Design](https://github.com/Chimoneg27/Awesome-Books/assets/43528594/192b9339-b383-4ff0-98c2-25da30e0b1c5)


</td>
      <td>

<img width="437" alt="awesome_books_core_elements" src="https://github.com/Chimoneg27/Awesome-Books/assets/43528594/a70b3a9d-e19d-4df6-8a7d-9883cc08202f">

</td>
    </tr>
  </table>